java/src/hu/holyoil/controller/AIController.java

```diff
@@ -35,7 +36,13 @@
      */
     @Override
     public void Step()  {
-        System.out.println("Stepping");
+        Logger.Log(this, "Steps" + (Main.isRandomEnabled ? "" : " - switched off!"));
+
+        if (Main.isRandomEnabled) {
```

Comment:
Randomness disabling should be in the classes themselves

Reply:
Well, we discussed, we don't want the AI to handle Ufo and Robot at all, if the disable_random is called.
So I moved the conditional to the Handle methods. But right now the teleportgates have this conditional too. What do I do with that? Stays liek that?